### PR TITLE
[FEATURE] header util for handling title, breadcrumb and urls of orders

### PR DESCRIPTION
**Pull request by turbo-ele**
**Tasks:**
- [x] create header util for setting title and breadcrumb from different bundles
- [x] get url method 
- [x] gather feedback for my changes

**Informations:**

| Q | A |
| --- | --- |
| Tests pass? | of course |
| Fixed tickets | - |
| BC Breaks | - |
| Doc | - |

### CODE REVIEW

File: Resources/public/js/util/header.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        var title = this.sandbox.translate('salesorder.order'),
+            breadcrumb = [
+                {title: 'navigation.sales'},
+                {title: 'salesorder.orders.title', event: 'sulu.salesorder.orders.list'}
+            ],
+        // parse options
+            hasOptions = typeof options === 'object',
+            titleAddition = (hasOptions && options.hasOwnProperty('titleAddition')) ? options.titleAddition : null,
+            breadcrumbAddition = (hasOptions && options.hasOwnProperty('breadcrumbAddition')) ? options.breadcrumbAddition : null,
+            orderEvent = (hasOptions && options.hasOwnProperty('breadcrumbOrderEvent')) ? options.breadcrumbOrderEvent : null;
```
Comment (notsure) [inline Resources/public/js/util/header.js]: I know these three lines above are pretty fancy as the if cause is written as a one liner .. but it is pretty hard to read and maintain. you might should consider writing it in a proper if cause and set the value default to null.

eg.:

var titleAddition = null;
if (hasOptions && options.hasOwnProperty('titleAddition')) {
titleAddtion = options.titleAddition;
}

less fancy but readable and not longer than 80 chars

File: Resources/public/js/util/header.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        /**
+         * sets header data: breadcrumb, headline for an order
+         * @param {Object} order Backbone-Entity
+         * @param {Object} options configuration object for options
+         * @param {String} [options.titleAddition] adds an extra text to the title
+         * @param {Array} [options.breadcrumbAddition] adds extra items to breadcrumb
+         * @param {String} [options.breadcrumbOrderEvent] event thats added to breadcrumb of the current order
+         */
+        setHeader: function(order, options) {
+
```
Comment (notsure) [inline Resources/public/js/util/header.js]: remove blank line

File: Resources/public/js/util/header.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        /**
+         * will create the url string for an order
+         * @param [id] if defined, 'edit:id' will be added to the url string
+         * @param [postfix] adds an url postfix
+         * @returns {string}
+         */
+        getUrl: function(id, postfix) {
+            var url = 'sales/orders';
+            if (!!id) {
+                 url += '/edit:' + id;
```
Comment (notsure) [inline Resources/public/js/util/header.js]: indent

File: Resources/public/js/util/header.js (hunk 1 above)
Comment (turbo-ele) [inline Resources/public/js/util/header.js]: lol

**Pull request merged**